geomstats/_backend/pytorch/__init__.py

```diff
@@ -88,6 +90,29 @@ def wrapper(x):
 sinh = _box_scalar(sinh)
 
 
+def to_numpy(x):
+    return x.numpy()
+
+
+def convert_to_wider_dtype(tensor_list):
+    dtype_list = [x.dtype for x in tensor_list]
+    wider_dtype = dtype_list[0]
+    wider_dtype_index = DTYPES.index(wider_dtype)
+    for dtype in dtype_list[1:]:
+        index = DTYPES.index(dtype)
+        if index > wider_dtype_index:
+            wider_dtype = dtype
+            wider_dtype_index = index
+    tensor_list = [cast(x, dtype=wider_dtype) for x in tensor_list]
+    return tensor_list
+
+
+def ceil(x):
```

Comment:
well spotted! What about using the `_box_scalar` wrapper though?

Reply:
I had forgotten about that decorator, thank you!